frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -293,11 +293,14 @@ export const Dashboard = () => {
             horizontal: 'right',
           }}
         >
-          <CSVLink data={csvCourses} filename={`export-courses-${Date.now()}`}>
+          <CSVLink
```

Comment:
I really like how concisely you were able to make this fix + efficiently used available functions to minimize the disruption to existing code. This works very well, based off what you demoed during work session yesterday. Great work!